appeals/e2e/cypress/page_objects/caseDetailsPage.js

```diff
@@ -195,13 +205,9 @@ export class CaseDetailsPage extends Page {
 	}
 
 	verifyChangeStartDate() {
-		const dateToday = new Date(); // Note: Months are 0-indexed, so 10 is November
-		const formattedDate = new Intl.DateTimeFormat('en-GB', {
-			day: 'numeric',
-			month: 'long',
-			year: 'numeric'
-		}).format(dateToday); // Format the date
-		cy.get('.appeal-start-date > .govuk-summary-list__value')
+		const formattedDate = dateTimeSection.formatDate();
```

Comment:
You'll need to leave in the `const dateToday = new Date();` line and then pass the `dateToday` variable into line 208, so it'll look like

```
const dateToday = new Date();
const formattedDate = dateTimeSection.formatDate(dateToday);
```

Reply:
Thanks, have now updated